From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: James Zern <jzern@chromium.org>
Date: Fri, 8 Sep 2023 20:57:18 +0000
Subject: Roll src/third_party/libwebp/src/ 6a319d4da..4619a48fc (1 commit)

https://chromium.googlesource.com/webm/libwebp.git/+log/6a319d4da395..4619a48fc329

$ git log 6a319d4da..4619a48fc --date=short --no-merges --format='%ad %ae %s'
2023-09-07 vrabaud Fix OOB write in BuildHuffmanTable.

Created with:
  roll-dep src/third_party/libwebp/src

Bug: 1479274
Change-Id: I0de56a278cfc64467d092ea0213bdf957b227ce1
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4853057
Commit-Queue: James Zern <jzern@google.com>
Reviewed-by: Urvang Joshi <urvang@chromium.org>
Cr-Commit-Position: refs/branch-heads/5845@{#1779}
Cr-Branched-From: 5a5dff63a4a4c63b9b18589819bebb2566c85443-refs/heads/main@{#1160321}

diff --git a/DEPS b/DEPS
index f2921b29c1bdc735f0be27951a9d3c99e16478f7..b3084dbc4cb522289edf698067995f1faff89610 100644
--- a/DEPS
+++ b/DEPS
@@ -1591,7 +1591,7 @@ deps = {
     Var('chromium_git') + '/webm/libwebm.git' + '@' + 'e4fbea0c9751ae8aa86629b197a28d8276a2b0da',
 
   'src/third_party/libwebp/src':
-    Var('chromium_git') + '/webm/libwebp.git' + '@' +  '6a319d4da3956f22f0dd5ad5a7bb294f88131bbd',
+    Var('chromium_git') + '/webm/libwebp.git' + '@' +  '4619a48fc3292743d7ce9658bee4245406734109',
 
   'src/third_party/libyuv':
     Var('chromium_git') + '/libyuv/libyuv.git' + '@' + '04821d1e7d60845525e8db55c7bcd41ef5be9406',
diff --git a/third_party/libwebp/README.chromium b/third_party/libwebp/README.chromium
index 6b51fa4a95340289f0549e53e52c85a2d17f5c37..0e24a4513b478c0e923eae3febafddc4916f1ce8 100644
--- a/third_party/libwebp/README.chromium
+++ b/third_party/libwebp/README.chromium
@@ -2,7 +2,7 @@ Name: WebP image encoder/decoder
 Short Name: libwebp
 URL: https://chromium.googlesource.com/webm/libwebp
 Version: 0
-Revision: 6a319d4da3956f22f0dd5ad5a7bb294f88131bbd
+Revision: 4619a48fc3292743d7ce9658bee4245406734109
 CPEPrefix: cpe:/a:webmproject:libwebp:1.3.0
 License: BSD
 License File: LICENSE
